### PR TITLE
Package bundling and new barcode types were added

### PR DESCRIPTION
**Pull request by ilguzin**
- Webpack has been engaged into the distribution process. Updated the sample layout code.
- Added support for "Code11" and "CodeMSI"

### CODE REVIEW

File: src/DecoderWorker.js [hunk 1]
```diff
@@ -877,6 +887,27 @@ var decoderWorkerBlob = function decoderWorkerBlob(){
...
     }
+    if (type === "Code11") {
+      totalBars = 5;
+      //len = binaryString[0];
+      //for(var i = 0; i < binaryString.length; i++) {
+      //  if(binaryString[i] > len*5) {
+      //    binaryString.splice(i, binaryString.length);
+      //    break;
+      //  }
+      //}
```
Comment (seangates) [inline src/DecoderWorker.js]: Remove comments if not used.

File: src/DecoderWorker.js [hunk 2]
```diff
@@ -919,14 +950,19 @@ var decoderWorkerBlob = function decoderWorkerBlob(){
           second = binaryString.splice(binaryString.length - 1, 1)[0];
         }
         binTemp = Distribution(binaryString, FormatPriority[i]);
+        if (FormatPriority[i] === "Code11") {}
+        if (FormatPriority[i] === "CodeMSI") {
+          //console.log("binaryString", binaryString);
+          //console.log("binTemp", binTemp);
+        }
```
Comment (seangates) [inline src/DecoderWorker.js]: Remove unused code.

File: src/DecoderWorker.js [hunk 3]
```diff
@@ -1509,6 +1588,19 @@ var decoderWorkerBlob = function decoderWorkerBlob(){
...
   }
 
+  function CheckCode11(string) {
+    var start = string[0].join("");
+    var end = string[string.length - 1].join("");
+    return Code11Encoding[start] === "SS" && Code11Encoding[end] === "SS";
+  }
+
+  function CheckCodeMSI(string) {
+    // console.log("CheckCodeMSI", string);
```
Comment (seangates) [inline src/DecoderWorker.js]: Remove unnecessary code.

File: src/DecoderWorker.js [hunk 4]
```diff
@@ -1533,6 +1625,86 @@ var decoderWorkerBlob = function decoderWorkerBlob(){
...
+      if (typeof CodeMSIEncoding[string[i].join("")] === 'undefined') {
+        isValid = false;
+      } else {
+        result += CodeMSIEncoding[string[i].join("")];
+      }
+    }
+    // todo implement check mod 10
+    //if (isValid) {
+    //  var checkMod = checkMod10(result);
+    //}
```
Comment (seangates) [inline src/DecoderWorker.js]: Remove unused code.

File: src/DecoderWorker.js [hunk 5]
```diff
@@ -834,7 +844,7 @@ var decoderWorkerBlob = function decoderWorkerBlob(){
         binTemp = [];
         for (i = 0; i < totalBars; i++) {
           binTemp.push(binaryString.splice(0, 1)[0]);
-          // binaryString.splice(0, 1)[0];
+          // binaryString.splice(0, 1)[0]; // todo uncomment?
```
Comment (seangates) [inline src/DecoderWorker.js]: Remove unused code.

**Review by seangates — CHANGES_REQUESTED** [review]
Fix these, and I'll see if I can get @EddieLa to merge this PR.

**Comment by ilguzin** [comment]
Sorry for not giving a feedback on this for so long. Accidentally found it to be still open :-)

**Pull request closed without merging**